modules/deploying-lvms-on-sno-cluster.adoc

```diff
@@ -25,11 +25,17 @@ You can deploy {lvms} using one of the following:
 [id="lvms-deployment-requirements-for-sno-ran_{context}"]
 == Requirements
 
-Before you begin deploying {lvms}, ensure that the following requirements are met:
+Before deploying {lvms} on {sno} clusters, ensure that the following requirements are met:
 
 * You have installed {rh-rhacm-first} on an {product-title} cluster.
 * Every managed cluster has dedicated disks that are used to provision storage.
 
+Before deploying {lvms} on a private CI environment, where virutal machines and storage devices can be reused, ensure that you have wiped the disks that are not in use. 
```

Comment:
Typo "virutal". Plus @agantony had mentioned an improvement that I wasn't sure if you saw/considered. I think it's a good suggestion that puts it more in active voice.

Posting here to fix the typo and use his suggestion, except putting it back to "wiped".

```suggestion
Before deploying {lvms} in a private CI environment where you can reuse virtual machines and storage devices, ensure that you have wiped the disks that are not in use.
```

Reply:
@bergerhoffer Thank you for pointing this out. I failed to notice this suggestion during the peer review. Updated the content based on the suggestion. PTAL.
Thanks.